src/components/OrganisationPublicView/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Banner from 'components/Organisation/Banner';
+import NavBar from 'components/NavBar';
+import { IOrganisation } from 'interfaces/organisation';
+import { FC } from 'react';
+import { useSelector } from 'react-redux';
+import { RootState } from 'reducers';
+import useFetchOrganisationOwnerManager from 'hooks/useFetchOrganisationOwnerManager';
+import styles from './index.module.scss';
+import HorizontalTabs from './HorizontalTabs';
+import OverviewTab from './OverviewTab';
+import ProjectsTab from './ProjectsTab';
+import PeopleTab from './PeopleTab';
+import JobsTab from './JobsTab';
+
+interface IOrganisationPublicView {
+  organisation: IOrganisation;
+}
+
+const OrganisationPublicView: FC<IOrganisationPublicView> = ({
+  organisation,
+}) => {
+  const org = useSelector((state: RootState) => state.org);
+  const { members } = useFetchOrganisationOwnerManager(
+    organisation.organisationId.toString()
+  );
```

Comment:
If the component is on organisation page then can use `useParams()` and get the `orgId`. Its type is `string | undefined`.